pandas/tseries/tests/test_timeseries.py

```diff
@@ -3541,6 +3541,15 @@ def test_datetimeindex_accessors(self):
         self.assertEqual(dti.is_year_end[0], False)
         self.assertEqual(dti.is_year_end[364], True)
 
+        # GH 11128
+        self.assertEqual(dti.weekday_name[4], u'Monday')
+        self.assertEqual(dti.weekday_name[5], u'Tuesday')
```

Comment:
can you add a test for ``Timestamp`` (see where tests for ``.weekday`` are, prob in ``tseries/test_timeseries.py``